src/Mvc/Mvc.NewtonsoftJson/src/NewtonsoftJsonOutputFormatter.cs

```diff
@@ -111,6 +111,13 @@ public override async Task WriteResponseBodyAsync(OutputFormatterWriteContext co
                 throw new ArgumentNullException(nameof(selectedEncoding));
             }
 
+            // Opt into sync IO support until we can work out an alternative https://github.com/aspnet/AspNetCore/issues/6397
+            var syncIOFeature = context.HttpContext.Features.Get<Http.Features.IHttpBodyControlFeature>();
```

Comment:
Same thing. This was a change in master, why is it showing up here in the diff?

Reply:
GitHub sometimes gets confused.